numpy/core/tests/test_umath.py

```diff
@@ -1403,14 +1403,28 @@ def __array_wrap__(self, arr, context):
                 return r
 
         a = with_wrap()
+        expected_args = (a, a, None)
+
+        def check_it(x):
+            assert_equal(x.arr, np.zeros(1))
+            func, args, i = x.context
+            assert_(func is ncu.minimum)
+            assert_equal(args, expected_args)
+            assert_equal(i, 0)
+
+
+
         x = ncu.minimum(a, a)
```

Comment:
I don't understand why this has changed. I think one should continue to suppress all-`None` outputs.

Reply:
All none outputs were not suppressed before when none was passed explicitly.

Reply:
But here you do not pass in `None`!

Reply:
Right, but it's important that passing the documented default behave the same as not passing it, whatever that behavior may be. I could get behind stripping all-none outputs too though - it just means users are far less likely to implement some-none outputs correctly